lib/cluster.js

```diff
@@ -196,12 +209,38 @@ function addListeners() {
 		// Respond to master's requests for worker's local metrics.
 		process.on('message', message => {
 			if (message.type === GET_METRICS_REQ) {
+				let workerRequests = 0;
+				workersQueue.forEach(worker => {
+					if (worker && worker instanceof Worker) {
```

Comment:
```suggestion
					if (worker instanceof Worker) {
```

Reply:
should we check this when instantiating/pushing into `workersQueue` rather than when receiving messages? We should probably throw